buildmodel/buildassets/buildassets_test.go

```diff
@@ -44,8 +63,10 @@ func TestBuildResultsDirectoryInfo_CreateSummary(t *testing.T) {
 		t.Errorf("CreateSummary() error is not wanted: %v", err)
 		return
 	}
-	if !reflect.DeepEqual(got, want) {
-		t.Errorf("CreateSummary() got = %v, want %v", got, want)
+	if diff := deep.Equal(got, want); diff != nil {
+		for _, d := range diff {
```

Comment:
The loop is not necessary to pretty print `diff`. You can pass it directly to `t.Error`.

Unrelated: I'm a big fan of `go-tets/deep` 😄

Reply:
Convinced me 😸